rankers/AggregateRanker/__init__.py

```diff
@@ -0,0 +1,32 @@
+from jina.executors.rankers import Chunk2DocRanker
+import numpy as np
+
+class AggregateRanker(Chunk2DocRanker):
+    """
+    :class:`AggregateRanker` calculates the score of the matched doc form the matched chunks. For each matched doc, the
+    score is aggregated from all the matched chunks belonging to that doc.
+    """
+
+    AGGREGATE_FUNCTIONS = ['min', 'max', 'mean', 'median', 'sum', 'prod']
+
+    def __init__(self, aggregate_function: str, is_reversed_score, *args, **kwargs):
```

Comment:
Please add a type hint and a reasonable default for `is_reserved_score`.

Reply:
Thanks, I added the type hint. Having a reasonable default for `is_reversed_score` is quite difficult. It depends on the used metric the chunks are compared with. If the chunk score is a distance function `is_reversed_score` should be set to `True` and if the chunk score is a similarity function `is_reversed_score` should be set to `False`.
If the ranker could somehow retrieve the type (similarity or distance) from the flow context, we would not even need that parameter.